react/locales/es-AR.json

```diff
@@ -1,5 +1,7 @@
 {
-  "impersonate-customer.message": "Conectado como",
-  "impersonate-customer.button": "Entrar como",
-  "impersonate-customer-logout.button": "Salir"
+  "telemarketing.attendant": "Asistente",
+  "telemarketing.client": "Cliente",
+  "telemarketing-login.message": "Entrar como cliente",
```

Comment:
Change those strings to es-AR, e.g: https://www.mercadolibre.com.ar/

Reply:
👍